.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: HTTPStan
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  tests:
+    name: HTTPStan tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-11.0, macos-10.15]
+        python-version: [3.7, 3.8, 3.9]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install poetry
+        run: |
+          pip install -U pip
+          pip install wheel
```

Comment:
I'm confused about `wheel` being installed here. It's not installed in .travis.yml.

Since we're all learning this new CI configuration language, I think having a clean translation would be helpful.

Reply:
Yes, the wheel was for pybind11 (it builds it from source).